### PR TITLE
backupccl: add 'aws-weekly' tag to restore/tpce/32tb test

### PR DESCRIPTION
**Pull request by msbutler**
Epic: none

Release note: None

### CODE REVIEW

**Review by healthy-pod — APPROVED** [review]
Do you want this test to run only in aws's weekly? If you want it on both gcp's and aws's weeklies you need keep both tags `weekly` and `aws-weekly`.

**Comment by msbutler** [comment]
note the small change I made to `test_registry.go` to prevent roachtest unit tests from failing.

**Comment by msbutler** [comment]
bors r=healthy-pod

**Comment by msbutler** [comment]
bors cancel

**Comment by renatolabs** [comment]
A little late to the party here, but can we consider keeping the `weekly` tag mandatory for every weekly test? It's convenient to be able to `list tag:weekly` to find out all tests that run weekly instead of nightly (it's way too easy to forget about `aws-weekly`).

The `Tags` field in `TestSpec` is a list, so we can tag this test with `[weekly, aws-weekly]`.

File: pkg/cmd/roachtest/test_registry.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func (r *testRegistryImpl) prepareSpec(spec *registry.TestSpec) error {
 	if spec.Timeout > maxTimeout {
 		var weekly bool
 		for _, tag := range spec.Tags {
-			if tag == "weekly" {
+			if tag == "weekly" || tag == "aws-weekly" {
```
[the remark this answers is not in the document]
Comment (msbutler) [reply pkg/cmd/roachtest/test_registry.go]: done.

**Review by renatolabs — APPROVED** [review]
Thank you!

**Comment by msbutler** [comment]
bors r=healthy-pod, renatolabs

**Pull request merged**